src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Views/Provider/StandardResultsPage.cs

```diff
@@ -1,42 +0,0 @@
-﻿using System.Collections.Generic;
-using FluentAssertions;
-using NUnit.Framework;
-using RazorGenerator.Testing; 
-using Sfa.Das.Sas.Web.UnitTests.Infrastructure.Web.ExtensionHelpers;
-using Sfa.Das.Sas.Web.ViewModels;
-using Sfa.Das.Sas.Web.Views.Provider;
-
-namespace Sfa.Das.Sas.Web.UnitTests.Infrastructure.Web.Views.Provider
-{
-    [TestFixture]
-    public class StandardResultsPage : ViewTestBase
-    {
-        [Test]
-        public void ShouldShowRegisterLink()
-        {
-            var detail = new StandardResults();
-            var model = new ProviderStandardSearchResultViewModel
-            {
-               IsLevyPayingEmployer = true,
-               Hits = new List<StandardProviderResultItemViewModel>()
-            };
-            var html = detail.RenderAsHtml(model).ToAngleSharp();
-
-            this.GetPartial(html, "#register_to_manage").Should().NotBeEmpty();
-        }
-
-        [Test]
-        public void ShouldNotShowRegisterLink()
-        {
-            var detail = new StandardResults();
-            var model = new ProviderStandardSearchResultViewModel
-            {
-                IsLevyPayingEmployer = false,
-                Hits = new List<StandardProviderResultItemViewModel>()
-            };
-            var html = detail.RenderAsHtml(model).ToAngleSharp();
-
-            this.GetPartial(html, "#register_to_manage").Should().BeEmpty();
-        }
-    }
-}
```

Comment:
Why did you remove these tests?

Reply:
Originally, I had repeated the html in the standardresults, Frameworkresults and Deatilas views. As they were in actual views, I could use AngleSharp to parse the whole document and write tests accordingly.

When I generalised them all into a partial view, I found that there wasn't a way for the HttpAgilityPack to render just that bit of HTML with the VM, and so I couldn't write tests for it.

Reply:
Fair enough!